tox.ini

```diff
@@ -5,4 +5,5 @@ deps = pytest==4.6.3
     mock==3.0.5
     pexpect==3.3
     pytest-vcr==1.0.2
+    cli_helpers[styles]==1.2.1
```

Comment:
In some newer version, the table's field padding rules change too, which causes test failures in `formatting`.